### PR TITLE
Added mdacli post

### LINKED ISSUE #204: inaugural mdacli blog post
Blog post to announce and advertise the first command line interface to MDAnalysis, https://github.com/MDAnalysis/mdacli

### PR DESCRIPTION
**Pull request by PicoCentauri**
Fixes #204 

The initial post for the first version of mdacli. We have to wait until we release the first version on PYPI. I am open for comments and improvements if something is unclear.

### CODE REVIEW

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thanks, already has all the information.

I ask myself what the intention of the blog post is. I see it mainly to *advertise* and get people to try mdacli out, so if this is your intention, too, then it should be allowing potential users faster to decide if this is for them.  I'd try to make it a bit more focused and roughly follow 
1. mdacli : name, synopsis, repository, doc links (I'd like the docs to be under mdanalysis.org, see https://github.com/MDAnalysis/mdacli/issues/71 )
2. *short* motivation (still link to Taurenmd and MAICOS)
3. *one line* installation, link to docs for more
4. *examples:* what can you actually do. I would show two examples, such as RMSD and RDF. Images would be neat because people always look at picture. I'd love an animated gif that shows the whole command line process. The examples will typically decide if people will try it or not.
5. link to list of tools that are already included
5. contributing
6. anything else (technical details, philosophy, ...) — link to docs for more

I would also include a logo.

See inline for minor comments.

File: _posts/2021-11-16-mdacli.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+scripts and face the same challenges and problems i.e.
+
+* How to initialize the universe and loop through frames without copying many lines of code?
+* How to write a CLI parser to analyze several of their simulations?
+* How to process and save their trajectories in a clever way?
+* ...
+
+Some of these problems can be solved by using the
+`MDAnalysis.analysis.base.AnalysisBase` class of MDA. However,
+this class is limited to python, and sometimes a direct CLI to these
```
[the remark this answers is not in the document]
Comment (PicoCentauri) [reply _posts/2021-11-16-mdacli.md]: Done.

File: _posts/2021-11-16-mdacli.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+New students/scientists start writing their analysis
+scripts and face the same challenges and problems i.e.
+
+* How to initialize the universe and loop through frames without copying many lines of code?
+* How to write a CLI parser to analyze several of their simulations?
+* How to process and save their trajectories in a clever way?
+* ...
+
+Some of these problems can be solved by using the
+`MDAnalysis.analysis.base.AnalysisBase` class of MDA. However,
```
[the remark this answers is not in the document]
Comment (PicoCentauri) [reply _posts/2021-11-16-mdacli.md]: Good point. I added the link.

File: _posts/2021-11-16-mdacli.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+project that uses the `MDAnalysis.analysis.base.AnalysisBase`
+as parent class. If new classes are added to the
+MDA codebase they are just they will show up without
+any adjustments to `mdacli` itself.
+
+The core of the wrapper is a docstring parser in combination
+with an argument inspection using the `inspect` library. Based on
+a created dictionary containing
+each parameter of the class with its docstring and type, the actual command-line interface is built using [argparse][argparse]. The syntax of the
+[GROMACS CLI][gmx_cli] syntax.
```
[the remark this answers is not in the document]
Comment (PicoCentauri) [reply _posts/2021-11-16-mdacli.md]: Indeed. I shortened the section notably.

File: _posts/2021-11-16-mdacli.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+layout: post
+title: "MDAnalysis Command Line Interface (CLI)"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdacli-logo.png"
+title="MDACLi LOGO"
+alt="The Logo of the MDAnlysis command line interface"
+style="float: right; height: 12em; " />
```
Comment (orbeckst) [inline _posts/2021-11-16-mdacli.md:11]: Logo is pretty big, maybe reduce size?

File: _posts/2021-11-16-mdacli.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+This command provides a list of all available modules similar to the 
+in the [documentation][documentation].
+The synopsis for calculating a radial distribution function (RDF) 
+between two groups is given by
+
+    mda interrdf -h
+
+A sample water trajectory is provided of rigid SPC/E water is
+provided [testfiles][online].
```
Comment (orbeckst) [inline _posts/2021-11-16-mdacli.md:56]: broken link
```suggestion
provided [testfiles][testfiles].
```

File: _posts/2021-11-16-mdacli.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+-- @PicoCentauri and @joaomcteixeira
+
+
+[taurenmd]: https://github.com/joaomcteixeira/taurenmd
+[maicos]: https://gitlab.com/maicos-devel/maicos
+[gmx_cli]: https://manual.gromacs.org/documentation/current/user-guide/cmdline.html
+[argparse]: https://docs.python.org/3/library/argparse.html
+[testfiles]: https://github.com/MDAnalysis/mdacli/tree/main/data
+[documentation]: https://mdacli.readthedocs.io/en/latest/
```
Comment (orbeckst) [inline _posts/2021-11-16-mdacli.md:122]: Change to https://www.mdanalysis.org/mdacli/ before publishing the post?

**Review by orbeckst — CHANGES_REQUESTED** [review]
It would be neater if the docs were available in the canonical spot when you publish the announcement.

Minor things see comments.

Change the release date to a few days into the future (rename the file).

File: _posts/2021-11-16-mdacli.md (hunk 4 above)
Comment (PicoCentauri) [reply _posts/2021-11-16-mdacli.md]: Yes!

File: _posts/2021-11-16-mdacli.md (hunk 6 above)
Comment (PicoCentauri) [reply _posts/2021-11-16-mdacli.md]: I changed it and we wait with the post until we published the docs on the webpage!

**Review by orbeckst — CHANGES_REQUESTED** [review]
Looks great.

I am just blocking the merge until https://github.com/MDAnalysis/mdacli/issues/71 (docs on mdanalysis.org) is done so that there are no broken links.

**Comment by orbeckst** [comment]
Are we now waiting for https://github.com/MDAnalysis/mdacli/pull/78 ?

**Comment by PicoCentauri** [comment]
If the docs are deployed we are ready to go :)

**Review by orbeckst — APPROVED** [review]

**Pull request merged**